scripts/publish-to-npm

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Sorry, I might be confused...Doesn't `yarn publish` by default pick the version from the package.json?

Reply:
You're right -- I must have put this in after having published while the version string was left as `develop`.

I also just discovered [the publishConfig setting for package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#publishconfig), rendering the other purpose of this script obsolete.